base/loading.jl

```diff
@@ -1078,6 +1078,7 @@ const pkgorigins = Dict{PkgId,PkgOrigin}()
 
 require(uuidkey::PkgId) = @lock require_lock _require_prelocked(uuidkey)
 
+const already_warned_path_change_pkgs = Set{String}()
```

Comment:
Can we use the `UUID` instead of a `String`?

Reply:
Yes, that's better.